app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons = []
+    for person_data in people:
+        name = person_data["name"]
+        age = person_data["age"]
+        person = Person(name, age)
+        persons.append(person)
+
+    for person_data in people:
+        person = Person.people[person_data["name"]]
+        if "wife" in person_data and person_data["wife"]:
+            setattr(person, "wife", Person.people[person_data["wife"]])
+        if "husband" in person_data and person_data["husband"]:
+            setattr(person, "husband", Person.people[person_data["husband"]])
```

Comment:
Similar to the 'wife' attribute, ensure that the 'husband' attribute is set only if the referenced person exists in `Person.people` to avoid potential `KeyError`.